src/components/ui/calendar/rangeCalendar.component.tsx

```diff
@@ -74,6 +74,10 @@ export type RangeCalendarElement<D = Date> = React.ReactElement<RangeCalendarPro
  * to render instead of default year cell.
  * Called with a date for this cell and styles provided by Eva.
  *
+ * @property {(D) => void} onVisibleDateChange - Called when
+ * dates in visible date selection component changed. You should only focus on
+ * `newDate`'s year and month, date is not useful.
+ *
```

Comment:
I believe this will be more clear :)
```suggestion
 * @property {(D) => void} onVisibleDateChange - Called when navigating to the previous or next month / year.
 *
```